install.md

```diff
@@ -2,6 +2,72 @@
 
 # Installation Instructions
 
+## Installing packaged versions of buildah
+
+### [Arch Linux](https://www.archlinux.org)
+
+```bash
+sudo pacman -S buildah
+```
+
+### [Fedora](https://www.fedoraproject.org), [CentOS](https://www.centos.org)
+
+```bash
+sudo yum -y install buildah
+```
+
+
+### [Fedora-CoreOS](https://coreos.fedoraproject.org), [Fedora SilverBlue](https://silverblue.fedoraproject.org)
+
+Not Available.  Must be installed via package layering.
```

Comment:
buildah is in silverblue by default: 
https://pagure.io/workstation-ostree-config/blob/31ad6acced8426a374d5af3701d0cfdbf58cb3da/f/fedora-common-ostree.json#_10

Reply:
Fixed